### PR TITLE
Add Run preupgrade button on job invocation page

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: lib/foreman_leapp/engine.rb [hunk 1]
```diff
@@ -81,11 +81,16 @@ class Engine < ::Rails::Engine
...
+    initializer 'foreman_leapp.require_dynflow',
                 before: 'foreman_tasks.initialize_dynflow',
                 after: 'foreman_remote_execution.require_dynflow' do |_app|
       ForemanTasks.dynflow.require!
       ForemanTasks.dynflow.config.eager_load_paths << File.join(ForemanLeapp::Engine.root, 'app/lib/actions')
     end
+
+    initializer('foreman_leapp.extend_remote_execution',
+                after: 'foreman_leapp.require_foreman_remote_execution') do |_app|
+      RemoteExecutionHelper.prepend ForemanLeapp::RemoteExecutionHelperExtension
```
Comment (adamruzicka) [inline lib/foreman_leapp/engine.rb:93]: Why not do this where all the other extensions happen https://github.com/oamg/foreman_leapp/pull/29/files#diff-8d68f56fa2177f1d565d390c12f94e3eR43-L45 ?

Comment (stejskalleos) [reply lib/foreman_leapp/engine.rb:93]: I tried it but it didn't work. What I tried:
```
config.to_prepare do
  RemoteExecutionHelper.prepend ForemanLeapp::RemoteExecutionHelperExtension
end
```
 `RemoteExecutionHelperExtension::job_invocation_task_buttons` was never called. This is the only way that worked for me.

**Comment by adamruzicka** [comment]
Wouldn't it be better if the button appeared next to other rerun buttons? Currently it appears on the other side

File: lib/foreman_leapp/engine.rb (hunk 1 above)
Comment (adamruzicka) [reply lib/foreman_leapp/engine.rb:93]: Huh, interesting. Nevermind then

**Comment by stejskalleos** [comment]
> Wouldn't it be better if the button appeared next to other rerun buttons? Currently it appears on the other side

Yeah that make sense, updated.

**Comment by adamruzicka** [comment]
Thank you @stejskalleos !

**Pull request merged**